src/tree/containerizedFunctionApp/FunctionTreeItem.ts

```diff
@@ -0,0 +1,54 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { createWebSiteClient } from "@microsoft/vscode-azext-azureappservice";
+import { DialogResponses, nonNullValueAndProp, type IActionContext } from "@microsoft/vscode-azext-utils";
+import { type ViewPropertiesModel } from "@microsoft/vscode-azureresources-api";
+import { ProgressLocation, window } from "vscode";
+import { ext } from "../../extensionVariables";
+import { localize } from "../../localize";
+import { FunctionTreeItemBase } from "../FunctionTreeItemBase";
+import { type FunctionItem } from "./FunctionItem";
+import { type FunctionsTreeItem } from "./FunctionsTreeItem";
+
+export class FunctionTreeItem extends FunctionTreeItemBase {
```

Comment:
We should name this something like `ContainerFunctionTreeItem` or something to help identify what it is. 

Of course, you can look at the folder name and figure that out, but I think it's still helpful to specify so we can tell immediately. `LocalFunctionTreeItem` and `RemoteFunctionTreeItem` do the same thing.